tests/functional_tests/ffront_tests/test_program.py

```diff
@@ -268,9 +268,18 @@ def test_copy_restrict_parsing(copy_restrict_program_def):
     pattern_node.match(past_node, raise_exception=True)
 
 
+function_definitions = [
+    itir.FunctionDefinition(
+        id="identity",
+        params=[itir.Sym(id="x")],
+        expr=itir.FunCall(fun=itir.SymRef(id="deref"), args=[itir.SymRef(id="x")]),
+    )
+]
```

Comment:
```suggestion
@pytest.fixture
def identity_itir_definition():
    return itir.FunctionDefinition(
        id="identity",
        params=[itir.Sym(id="x")],
        expr=itir.FunCall(fun=itir.SymRef(id="deref"), args=[itir.SymRef(id="x")]),
    )
```
I think this should be a fixture at the top of the file.